src/main/java/org/cyclonedx/generators/json/AbstractBomJsonGenerator.java

```diff
@@ -94,6 +108,109 @@ void createComponentsNode(final JSONObject json, final List<Component> component
         }
     }
 
+    void createExtensionsNode(final JSONObject json, Map<String, Extension> extensions){
+        if (extensions != null) {
+            for (Map.Entry<String, Extension> extension : extensions.entrySet()) {
+                if (ExtensionType.VULNERABILITIES.getTypeName().equals(extension.getKey())) {
+                    processVulnerabilities(json, extension.getValue(), extension.getKey());
+                }
+            }
+        }
+    }
+
+    private void processVulnerabilities(final JSONObject json, final Extension extension, final String nodeName){
+        if (extension != null && extension.getExtensions() != null && !extension.getExtensions().isEmpty()) {
+            final JSONArray vulnArray = new JSONArray();
+            for (ExtensibleType ext : extension.getExtensions()) {
+                final Vulnerability1_0 vuln = (Vulnerability1_0) ext;
```

Comment:
Q: should we check if the `ext` is an `instanceof Vulnerability1_0`  before casting here?